app/namespace.go

```diff
@@ -57,6 +60,38 @@ var (
 		"us-east-1",
 		"us-west-2",
 	}
+	sinkNameFlag = &cli.StringFlag{
+		Name:     "sink-name",
+		Usage:    "sink name",
```

Comment:
TODO: Probably need to enforce length and character limits on sink name on server side now that this is user assigned...